content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,179 @@
+  ([Issue #30013](https://github.com/istio/istio/issues/30013))
+
+- **Updated** non-revisioned installs to target the label `istio.io/rev=default` for injection in addition to the
+existing default injection labels (`istio-injection=enabled` and `sidecar.istio.io/inject=true`).
+
+- **Added** support for slash characters in environment variables on `injectionURL`.
+  ([Issue #31732](https://github.com/istio/istio/issues/31732))
+
+- **Added** an `external` profile for installing Istio with an external control plane and deprecated the `remote` profile.
+  ([Issue #32370](https://github.com/istio/istio/issues/32370))
+
+- **Fixed** a bug preventing `istioctl kube-inject` from working with revisions.
+  ([Issue #30991](https://github.com/istio/istio/issues/30991))
+
+- **Improved** the output of istioctl YAML diff commands.
+  ([Issue #31186](https://github.com/istio/istio/issues/31186))
+
+- **Removed** the `15012` and `15443` ports from the default gateway installation. These can be explicitly
+[added](/docs/setup/install/istioctl/#configure-gateways) if desired, although it is
+recommended to follow the new [multicluster installation guide](/docs/setup/install/multicluster/) instead.
+
+- **Updated** Kiali addon to the latest version `v1.34`.
+
+## istioctl
+
+- **Updated** the `istioctl experimental precheck` command to identify potential upgrade issues prior to actually running an upgrade.
+
+- **Updated** `istioctl kube-inject` to call webhook server to get the injection template by default.
+  ([Issue #29270](https://github.com/istio/istio/issues/29270))
+
+- **Added** `istioctl experimental internal-debug` to retrieve istiod debug information via a secured debug interface.
+  ([Issue #31338](https://github.com/istio/istio/issues/31338))
+
+- **Added** `istioctl validate` and the validating webhook now report duplicate or unreachable virtual service matches.
+  ([Issue #31525](https://github.com/istio/istio/issues/31525))
+
+- **Added** `istioctl proxy-config -o yaml` to display in YAML along with the current JSON and short format.
+ ([Usage](/docs/reference/commands/istioctl/#istioctl-proxy-config), [Issue #31695](https://github.com/istio/istio/issues/31695))
+
+- **Added** the `istioctl proxy-config all` command to view the full proxy configuration.
+
+- **Added** tooling for revision-centric view of current Istio deployments in a cluster. This is to
+provide a better understanding of deployments- such as the number of istiod, gateway pods, `IstioOperator` custom resources-
+defining a particular revision, and the number of pods with sidecars pointing to a particular revision. ([Issue #23892](https://github.com/istio/istio/issues/23892))
+
+- **Added** a new analyzer for invalid webhook configurations.
+
+- **Fixed** fail correctly when `istioctl x create-remote-secret --secret-name` points to a non-existent secret in the remote cluster.  ([Issue #30723](https://github.com/istio/istio/issues/30723))
```

Comment:
Fixed an issue where `istioctl x create-remote-secret --secret-name` failed incorrectly when pointing  or something to fix the grammar at the front of the item.